### PR TITLE
Upgraded to latest socket.io version

### PR DESCRIPTION
**Pull request by jsilvestre**
This is a breaking version PR (be careful when bumping version number).

Each app has to be reviewed and updated manually depending on its needs and specificities (Files requires peculiar attention).

Tested against:
* Files
* Calendar
* Home

I will open PRs on all the applications once this one is merged and published.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "cozy-realtime-adapter",
   "version": "0.11.4",
   "description": "A simple binder from cozy-data-system's events to backbone models",
-  "main": "lib/server.js",
+  "main": "src/server.coffee",
```
Comment (aenario) [inline package.json]: why ?

File: package.json [hunk 2]
```diff
@@ -29,6 +30,6 @@
   "_resolved": "https://registry.npmjs.org/cozy-realtime-adapter/-/cozy-realtime-adapter-0.0.7.tgz",
   "devDependencies": {
     "express": "~3.2.5",
```
Comment (aenario) [inline package.json]: not yours, but please make all dependencies strict.

File: package.json (hunk 1 above)
Comment (jsilvestre) [inline package.json]: my mistake, was used conveniently with npm link

File: src/server.coffee [hunk 3]
```diff
@@ -9,44 +9,37 @@ path = require 'path'
...
+    options ?= {}
+    options.serveClient = true
+    server.io = sio server, options
 
+    # start axon's socket
     socket = axon.socket 'sub-emitter'
     socket.connect 9105
 
-    logging.log 'Realtime-adapter : socket.io initialized !'
+    log.info 'Realtime-Adapter: socket.io initialized'
```
Comment (aenario) [inline src/server.coffee]: prefix already in printit

**Comment by aenario** [comment]
my first npm version major \o/

**Comment by jsilvestre** [comment]
\o/

**Comment by aenario** [comment]
published as v1.0.0

**Comment by jsilvestre** [comment]
Thanks!

**Pull request merged**